Tests/ClientTests/Helpers/RatingPromptManagerTests.swift

```diff
@@ -103,9 +104,10 @@ class RatingPromptManagerTests: XCTestCase {
     }
 
     func testShouldShowPrompt_hasMinimumMobileBookmarksCount_returnsTrue() {
-        setupEnvironment()
-        createBookmarks(bookmarkCount: 5, withRoot: BookmarkRoots.MobileFolderGUID)
-        updateData(expectedRatingPromptOpenCount: 1)
+        _ = XCTSkip("flakey test")
```

Comment:
I think I saw some tasks created for some of the disabled tests? If so we can add them as a comment in the XCTSkip, easier to keep track this way.

Reply:
I didn't add one for site image helper. I figured there is no point spending time updating the tests when we are actively working on deprecating and removing that class anyway.

Reply:
Yeah make sense !